dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -174,28 +247,44 @@ export const Columns: React.FC<{
 			)}
 
 			<li role="none">
-				<ThemeProvider theme={{ ...buttonThemeBrand }}>
-					<div css={searchBar}>
-						<LinkButton
-							href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-							tabIndex={-1}
-							className="selectableMenuItem"
-							priority="secondary"
-							icon={
-								<SvgMagnifyingGlass
-									isAnnouncedByScreenReader={true}
-									size="medium"
-								/>
-							}
-							aria-label="Search with google"
-							data-link-name="nav2 : search : submit"
-							type="submit"
-						>
-							Search
-						</LinkButton>
-					</div>
-				</ThemeProvider>
+				<form css={searchBar} action="https://www.google.co.uk/search">
+					<TextInput
+						hideLabel={true}
+						label="Search input"
+						cssOverrides={searchInput}
+						name="q"
+						placeholder="Search"
+						data-link-name="nav2 : search"
+						tabIndex={-1}
+					/>
 
+					<Label hideLabel={true} text="google-search">
+						<div css={searchGlass}>
+							<SvgMagnifyingGlass
+								isAnnouncedByScreenReader={true}
```

Comment:
What will the screen reader announce here? I'm not 100% sure how this works but my guess is it will look for the label text but it seems as though there's none here